frontend/src/components/collective/CollectiveExpenseItem.js

```diff
@@ -37,13 +37,15 @@ const CollectiveExpenseItem = ({
 
   const updateInProgress = approveInProgress[expense.id] || rejectInProgress[expense.id] || payInProgress[expense.id];
 
+  const userEmail = user && (user.paypalEmail || user.email);
```

Comment:
You are right.
I'm confused 🤔
Time to go to bed here.

Tests were failing. I've restarted them.